src/style.css

```diff
@@ -1,3 +1,81 @@
-body {
+html{
+  font-family: Roboto, Poppins, sans-serif
+}
+
+body{
+  margin: 0;
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
+*{
+  box-sizing: border-box;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: inline-block;
+  text-decoration: none;
+  margin: 0;
+  padding: 0;
+}
+
+.logo-images {
+  width: 40px;
+  height: 40px;
   margin: 0;
+  padding: 0;
+  border: 0;
+  display: block;
```

Comment:
it's redundant

Reply:
display block you can leave, or set width and height for .logo

Reply:
Since I completely remove the .logo according to your lower notes, I leave it as it is